### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [mcclaintech2011/f1-3-c2p1-colmar-academy#1]
Descriptive alt text should be done for each image.  Screen readers for the visually impaired uses alt text to describe the picture.
